src/state/connection-mixin.ts

```diff
@@ -26,6 +26,10 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
   superClass: T
 ) =>
   class extends superClass {
+    private __hiddenTimeout?: number;
```

Comment:
We shouldn't put this on the `connection-mixin`. This is re-used by different projects. Instead let's either add it to `home-assistant.ts` or a new mixin